kartothek/io_components/metapartition.py

```diff
@@ -1550,6 +1553,10 @@ def parse_input_to_metapartition(obj, metadata_version=None):
     obj : Union[Dict, pd.DataFrame, kartothek.io_components.metapartition.MetaPartition]
     metadata_version : int, optional
         The kartothek dataset specification version
+    expected_secondary_indices : Union[Set[str], Literal[False]], optional
```

Comment:
Any reason why to use `False` instead of `None` as the "don't verify anything" sentinel?

Reply:
IMHO, `None` may be intepreted as "ensure there are no secondary indices", whereas `False` seems more clearly to represent "don't verify anything"

Reply:
Both `None` and `False` are allowed now as "don't verify anything".